tests/assembly/nvptx-c-abi-ret-v7.rs

```diff
@@ -0,0 +1,230 @@
+//@ assembly-output: ptx-linker
+//@ compile-flags: --crate-type cdylib -C target-cpu=sm_86 -Z unstable-options -Clinker-flavor=llbc
+//@ only-nvptx64
+//@ ignore-nvptx64
```

Comment:
Why does this have to be ignored? You are still using `PassMode::Direct` for the return value, so I thought that would still generate the right ABI?

Reply:
It's unfortunately slightly broken (from before this PR) since there never were any tests for this. 

I will enable these tests in the follow up PR we talked about that I hopefully will be able to create today

Reply:
Enabled in #125980